src/installer.ts

```diff
@@ -338,10 +370,112 @@ async function resolveVersionFromManifest(
   }
 }
 
+function evaluateNightlyVersions(
+  versions: string[],
+  versionSpec: string
+): string {
+  let version = '';
+  let range: string | null | undefined;
```

Comment:
```suggestion
  let range: string | undefined;
```

doesn't look like it's ever `null`?